### PR TITLE
Improve HTML

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#367 
#### What does this implement/fix? Explain your changes.
- HTML report modified

#### Any other comments?
Validator: https://validator.w3.org/

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari

Hi folks ;-)

This PR is for #367, please check carefully and give me some feedback ;-)


**Notice 1**: Do not hurry, we have enough time ;-)
**Notice 2**: Please check `HTML` output and compare it with the previous version.

File: pycm/pycm_output.py [hunk 1]
```diff
@@ -15,14 +15,7 @@ def html_init(name):
...
-    result = ""
-    result += "<html>\n"
-    result += "<head>\n"
-    result += "<title>" + str(name) + "</title>\n"
-    result += "</head>\n"
-    result += "<body>\n"
-    result += '<h1 style="border-bottom:1px solid ' \
-              'black;text-align:center;">PyCM Report</h1>'
+    result = HTML_INIT_TEMPLATE.format(str(name), OG_DESCRIPTION, OG_IMAGE_URL)
     return result
```
Comment (sadrasabouri) [inline pycm/pycm_output.py:19]: You can also return it directly, like
```
return HTML_INIT_TEMPLATE.format(str(name), OG_DESCRIPTION, OG_IMAGE_URL)
```

File: pycm/pycm_output.py [hunk 2]
```diff
@@ -276,11 +269,7 @@ def html_end(version):
     :type version:str
     :return: html_end as str
     """
-    result = "</body>\n"
-    result += '<p style="text-align:center;border-top:1px solid black;">Generated By ' \
-        '<a href="http://www.pycm.ir" ' \
-              'style="text-decoration:none;color:red;">PyCM</a> Version ' + version + '</p>\n'
-    result += "</html>"
+    result = HTML_END_TEMPLATE.format(version)
     return result
```
Comment (sadrasabouri) [inline pycm/pycm_output.py:273]: Here too.

File: pycm/pycm_param.py [hunk 3]
```diff
@@ -18,6 +18,36 @@
...
 '''
 
+OG_IMAGE_URL = "http://www.pycm.ir/images/logo-og.png"
+
+OG_DESCRIPTION = "PyCM is a multi-class confusion matrix library written in Python. http://www.pycm.ir"
+
+HTML_INIT_TEMPLATE = '''
+<!doctype html>
+<html lang="en">
+<head>
```
Comment (sadrasabouri) [inline pycm/pycm_param.py:28]: There were a `\n` after `<head>` in latest `dev` version, is it removed on purpose?

File: pycm/pycm_output.py (hunk 2 above)
Comment (sepandhaghighi) [reply pycm/pycm_output.py]: Fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/fb72bbf1568c129a5e48284386c2aa66bc48ac1f

File: pycm/pycm_output.py (hunk 1 above)
Comment (sepandhaghighi) [reply pycm/pycm_output.py]: Fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/fb72bbf1568c129a5e48284386c2aa66bc48ac1f

File: pycm/pycm_param.py (hunk 3 above)
Comment (sepandhaghighi) [reply pycm/pycm_param.py:28]: No need to extra `\n` in a multi-line string

**Review by sadrasabouri — APPROVED** [review]
LGTM

File: pycm/pycm_param.py (hunk 3 above)
Comment (sepandhaghighi) [reply pycm/pycm_param.py:27]: fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/868923e82cc48767fb62fb9fcb61823fe031a432

**Comment by sepandhaghighi** [comment]
> LGTM

@sadrasabouri I made some changes (https://github.com/sepandhaghighi/pycm/pull/368/commits/868923e82cc48767fb62fb9fcb61823fe031a432, https://github.com/sepandhaghighi/pycm/pull/368/commits/1ebf8151aa68fc45b47d0ae81138caddeb92e25b, https://github.com/sepandhaghighi/pycm/pull/368/commits/9a0dbb0df7629553a33f8a3eba11347dd8a277ac).
Please review again 🔢

**Review by sadrasabouri — APPROVED** [review]

**Review by alirezazolanvari — APPROVED** [review]
Nice work! 👍

**Comment by sepandhaghighi** [comment]
> Nice work! 👍

@alirezazolanvari If everything is okay, please merge this PR 💯

**Pull request merged**